config/locales/en.yml

```diff
@@ -242,6 +242,41 @@ en:
               - "I cannot go out because I have a disability"
               - "I’m unable to leave my home for another reason"
   results_link:
+    feel_unsafe:
+      title: "Feeling Safe"
```

Comment:
We should be able to get this from the group.

Reply:
How do you mean?
Ideally I'd like to keep these as aligned to the keys as possible?

So i can ask the session for the relevant keys then look up the right part of the data structure?

It makes sense to me to pass in one data structure instead of do multiple lookups, if i get this group key out of this part of the YAML i get:
- The question title for the header,
- The key i can use to exclude groups we don't need
- The question headers we'll need for each section
- The result question critiera we can use to exclude questions we don't need
- the results to render

If i did it another way i'd need to splice all that together in the helper no?

Reply:
If the downside is it clutters the locale file,

I'm also generally thinking that i really am not worried about how clean this YAML file is (if it's still here in 6th months i'll regret typing this).

But there's that other branch around questions as data, that I think should really clean up and make this all a nicer place for content designers to work with. As there shouldn't be any updates over the weekend, we could prioritize that as first thing after MVP and tidy this up then?

Reply:
If you have the key name for the group, you can use it to lookup the group title from the locale file, rather than defining it twice and risking them getting out of sync.

Reply:
This will become clear when the page helper is written.

Reply:
ok removed